internal/uvm/counter.go

```diff
@@ -15,3 +15,9 @@ func (uvm *UtilityVM) ContainerCounter() uint64 {
 func (uvm *UtilityVM) UVMMountCounter() uint64 {
 	return atomic.AddUint64(&uvm.mountCounter, 1)
 }
+
+// layer counter is used for maintaining the number of scsi layers mounted
+// on the UVM
+func (uvm *UtilityVM) UVMLayerCounter() uint64 {
```

Comment:
I wonder if we should just go with a single counter for all SCSI mounts. Thoughts?

Reply:
The question is like do we want a directory that is specifically for layers or not since it doesn't make sense to use the same counter for content in different directories. I went with yes here to make things more explicit and for greater control on our end in the future. What are your thoughts?

Reply:
We already have a mountCounter in UVM struct right? It is used for similar purpose. Why can't we use the same counter?

Reply:
yes, I liked having the separation between the mounts and layers dirs for clarity, but honestly I can't think of a good use case for needing the separation. So I'll change to this. All layers and mounts will now be under the lcow global mount path.

Reply:
Done.